code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -566,7 +566,6 @@
 
 	//Projectiles with bonus SA damage
 	if(!Proj.nodamage)
-		var/true_damage = Proj.damage

```

Comment:
Removed this too. Var was not used. It throws a compile warning but that doesn't bother Travis, just noticed when compiling y'alls code.